gradle.properties

```diff
@@ -0,0 +1,9 @@
+# WARNING - THIS FILE WAS GENERATED BY THE dps-gradle-spring-boot GRADLE PLUGIN
+#           AND ANY MANUAL CHANGES WILL BE OVERRIDDEN ON YOUR NEXT BUILD.
+#
+# To make general changes to the configuration below, change the gradle plugin dps-gradle-spring-boot,
+# publish a new version and update to the new version in your gradle build script
+#
+# To stop the dps-gradle-spring-boot project from overwriting any project specific customisations here, remove the
+# warning at the top of this file.
+kotlin.incremental.useClasspathSnapshot=false
```

Comment:
whats this for?

Reply:
good spot @AledWEvans  :) this was a change triggered by the gradle spring boot update which was not checked in earlier.